packages/hash/api/src/model/ontology/data-type.model.ts

```diff
@@ -162,24 +149,29 @@ export default class {
    *   Depends on the RFC captured by:
    *   https://app.asana.com/0/1200211978612931/1202464168422955/f
    *
-   * @param params.accountId the accountId of the account making the update
    * @param params.schema a `DataType`
    */
   async update(
     graphApi: GraphApi,
     params: {
-      accountId: string;
       // we have to manually specify this type because of 'intended' limitations of `Omit` with extended Record types:
       //  https://github.com/microsoft/TypeScript/issues/50638
       //  this is needed for as long as DataType extends Record
       schema: Pick<DataType, "kind" | "title" | "description" | "type"> &
         Record<string, any>;
     },
   ): Promise<DataTypeModel> {
-    const { accountId, schema } = params;
+    const { schema } = params;
 
     const updateArguments: UpdateDataTypeRequest = {
-      accountId,
+      /**
+       * @todo: let caller update who owns the type, or create new method dedicated to changing the owner of the type
+       * @see https://app.asana.com/0/1202805690238892/1203063463721791/f
+       *
+       * @todo: replace uses of `accountId` with `ownedById` in the Graph API
+       * @see https://app.asana.com/0/1202805690238892/1203063463721791/f
```

Comment:
Should this asana task be addressed more towards the graph API than the model classes?

Reply:
It's a task for the Graph API, yes - should I not leave todos where we're waiting for tasks to be completed for the Graph API?

Reply:
Oh sorry, I read the task to be about the workspace API and not the graph API, my bad. I was implying we should change workspace to graph API in the task, but it is about that. I misread :sweat_smile: